fluent/protocol/handshake.go

```diff
@@ -149,7 +149,8 @@ func ValidatePongDigest(p *Pong, key, nonce, salt []byte) error {
 
 func validateDigest(received, key, nonce, salt []byte, hostname string) error {
 	expected := computeHexDigest(salt, hostname, nonce, key)
-	if bytes.Compare(received, expected) != 0 {
+	//if bytes.Compare(received, expected) != 0 {
+	if !bytes.Equal(received, expected) {
```

Comment:
you confident of this change?

Reply:
I'm mildly confident in it, this is the recommended change that the linter made.  Additionally, the documentation on https://pkg.go.dev/bytes#Equal implies these two ideas being the same and all unit tests regarding this area still pass.  Although, I do need to remove the commented out code.